seatunnel-e2e/pom.xml

```diff
@@ -34,4 +34,22 @@
         <module>seatunnel-flink-sql-e2e</module>
     </modules>
 
+    <dependencies>
```

Comment:
If we add all dist to e2e, @ruanwenjun Is that some bug can't be find? Like plugin search?

Reply:
I don't think add all dist to e2e will cause the plugin search bug can't not be found.

Reply:
We don't need to add connector-dist dependency into e2e, the e2e module find the connector jar from connector module and submit it to docker. If we add the connector-dist to e2e, we need to update the e2e base class, otherwise, the dependency is not used.

Reply:
We don't need to add connector-dist dependency into e2e, the e2e module find the connector jar from connector module and submit it to docker. If we add the connector-dist to e2e, we need to update the e2e base class, otherwise, the dependency is not used.

Reply:
> We don't need to add connector-dist dependency into e2e, the e2e module find the connector jar from connector module and submit it to docker. If we add the connector-dist to e2e, we need to update the e2e base class, otherwise, the dependency is not used.

I remove the dependency now.

Reply:
<img width="973" alt="image" src="https://user-images.githubusercontent.com/32193458/177141104-6b6fa6d7-f7be-4026-ac07-104b26b6e83f.png">

@ruanwenjun  This problem reappears after a remove connector-dist dependency from e2e. How can we control the build order about the modules?